### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by KevinBatdorf** [alpinejs/alpine#963] [4. General]
It's the end of an interesting year, and since Alpine and the community here has had a significant impact on my life in 2020 I just wanted to show some gratitude and say thank you to Caleb and everyone else. It's been nothing but good vibes all around.

**Comment by kylethielk** [comment]
Completely agree! Alpine fits the bill perfectly allowing us to sprinkle in some Reactivity easily without needing the complexity of a huge SPA driven framework. Looking forward to seeing Alpine gain even more popularity and community support.
